gallery/updatecompliance-insights/Azure Monitor.json

```diff
@@ -3,14 +3,14 @@
 	"version": "TemplateGallery/1.0",
 	"categories": [
 		{
-			"id": "Update Compliance",
-			"name": "Update Compliance",
+			"id": "Windows Update for Business reports",
+			"name": "Windows Update for Business reports",
 			"templates": [
 				{
-					"id": "Workbooks/UpdateCompliance/UpdateComplianceHub",
+					"id": "Workbooks/UpdateCompliance/Windows Update for Business reports",
```

Comment:
Can we leave the id the same? it'll affect telemetry and anyone who's pinned anything to their dashboards will get 404 since there's no redirects for workbook templates